src/pages/event.js

```diff
@@ -190,77 +218,71 @@ export function Event() {
+              <h2>{errorEvent || 'Token not found'}</h2>
+              <div >
+                <img alt="warning sign" style={{maxWidth: '30rem'}} src="/icons/warning.svg"/>
+              </div>
+            </div> :
+            <div className="container">
+              <div style={{
+                display: 'flex',
+                alignItems: 'center',
+                flexWrap: 'wrap',
+                alignContent: 'space-around',
+                justifyContent: 'space-around',
+                marginBottom: 82,
+              }}>
+                <div style={{flex: '0 0 18rem', display: 'flex', flexDirection: "column", justifyContent: "center"}}>
+                  <div className='prev-next-buttons' style={{display: 'flex', justifyContent: 'space-between', marginBottom: 38,}}>
+                    <a href={parseInt(eventId)-1} ><FontAwesomeIcon icon={faAngleLeft}/>{'  Prev'}</a>
+                    <h4 style={{marginBottom: '0'}}><div className='event-title'>EVENT ID</div><div className='event-id'>#{eventId}</div> </h4>
+                    <a href={parseInt(eventId)+1} >{'Next  '}<FontAwesomeIcon icon={faAngleRight}/></a>
+                  </div>
+                  <div style={{minHeight: '200px', margin: '0 auto'}}>
+                    <EventCard key={0} event={event} size='l' power={power} />
+                  </div>
+                </div>
+              </div>
+              <div className='table-header'>
+                <div className='table-title'>Collections <span>({tokens.length})</span></div>
+                {csvReadyOrAlmostReady() ?
+                  <CSVLink
+                      filename={`${event.name}.csv`}
+                      target="_blank"
+                      data-tip={`${csvOnlyMissingEns() ? 'Please wait if you want the ens names too' : ''}`}
+                      className={'btn'}
+                      style={csvButtonStyle}
+                      data={csv_data}
+                  >
+                    <CSVLinkText>{`Download CSV${csvOnlyMissingEns() ? ' (without ens)' : ''}`}</CSVLinkText>
+                    <ReactTooltip effect={'solid'} />
+                  </CSVLink> :
+                  <button
+                      className={'btn button-disabled'}
+                      style={csvButtonStyle}
+                      data-tip={'Please wait for the POAPs data to be loaded'}
+                      onClick={null}
+                  ><span style={{margin: 0}}>Download CSV</span><ReactTooltip effect={'solid'} /></button>
+                }
+              </div>
+              <div className='table-container'>
+                {
+                  width > 480
+                      ? <CreateTable event={event} loading={loadingEvent !== 'succeeded'} columns={columns} data={data} pageCount={pageCount} />
+                      : <CreateMobileTable event={event} loading={loadingEvent !== 'succeeded'} columns={mobileColumns} data={data} pageCount={pageCount} />
```

Comment:
If you see lines 278 and 279 they are practically identical. In this case we can use an adapter pattern for a more verbose code
 
```js
{   const isLoading = loadingEvent !== 'succeeded'
    const AdapterCreateTable = (width > 480) ? CreateTable : CreateMobileTable
    <AdapterCreateTable event={event} 
                        loading={isLoading} 
                        columns={mobileColumns} 
                        data={data} 
                        pageCount={pageCount} />
}    
```